ai-service/src/ai_service/handlers/answer.py

```
@@ -43,15 +43,24 @@
+                "- Answer based on what you can observe in the code, configurations, and documentation\n"
+                "- If you find relevant implementation details, explain how things work\n"
+                "- If you see file structures or directories, mention their organization\n"
+                "- When discussing features, explain what exists vs what doesn't exist\n"
+                "- Be specific about file locations when relevant\n"
+                "- If the context doesn't contain enough information, say so clearly\n"
+                "- Keep responses concise but informative\n\n"
+                "ANSWER:"
             )
             logger.info("User question: %s", user_question)
             logger.info("Most relevant code snippet(s): %s", context)
+            logger.info(f"Context length: {len(context)} characters")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)